submittal/src/main/java/com/kyriba/submittal/domain/Speaker.java

```diff
@@ -16,20 +27,29 @@
 /**
  * @author M-ABL
  */
+@Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = { "email" }))

```

Comment:
Useless if you not generate schema from mapping.

Reply:
Will remove it after migration to liquibase